zipkin-autoconfigure/ui/src/main/java/zipkin/autoconfigure/ui/ZipkinUiProperties.java

```diff
@@ -54,4 +55,12 @@ public String getInstrumented() {
   public void setInstrumented(String instrumented) {
     this.instrumented = instrumented;
   }
+
+  public String getLogsUrl() {
+    return logsUrl;
+  }
+
+  public void setLogsUrl(String logsUrl) {
```

Comment:
I'd test this in ZipkinUiAutoConfigurationTest basically that when `zipkin.ui.logs-url:` (empty string), `getLogsUrl()` is null, and when it is set to a url, that's readable as `getLogsUrl()`. This helps make sure there's no typos hard to see.